example/lib/test_scenarios/bluetooth_state_toggle_scenario.dart

```diff
@@ -22,6 +29,8 @@ class BluetoothStateTestScenario implements TestScenario {
 
     log("Enabled radio!");
 
+    log("Get radio state: ${bleManager.state()}");
```

Comment:
Won't this log `Future...` instead of actual state? Shouldn't there be an await?